pkg/agent/datamodel/defaults.go

```diff
@@ -389,20 +392,20 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 			// Distro assignment for masterProfile
 			if cs.Properties.MasterProfile.Distro == "" && cs.Properties.MasterProfile.ImageRef == nil {
 				if cs.Properties.OrchestratorProfile.IsKubernetes() && cs.Properties.OrchestratorProfile.KubernetesConfig.CustomHyperkubeImage == "" {
-					cs.Properties.MasterProfile.Distro = api.AKSUbuntu1604
+					cs.Properties.MasterProfile.Distro = AKSUbuntu1604
 				} else {
-					cs.Properties.MasterProfile.Distro = api.Ubuntu
+					cs.Properties.MasterProfile.Distro = Ubuntu
 				}
 			} else if cs.Properties.OrchestratorProfile.IsKubernetes() && (isUpgrade || isScale) {
-				if cs.Properties.MasterProfile.Distro == api.AKSDockerEngine || cs.Properties.MasterProfile.Distro == api.AKS1604Deprecated {
-					cs.Properties.MasterProfile.Distro = api.AKSUbuntu1604
-				} else if cs.Properties.MasterProfile.Distro == api.AKS1804Deprecated {
-					cs.Properties.MasterProfile.Distro = api.AKSUbuntu1804
+				if cs.Properties.MasterProfile.Distro == AKSDockerEngine || cs.Properties.MasterProfile.Distro == AKS1604Deprecated {
+					cs.Properties.MasterProfile.Distro = AKSUbuntu1604
+				} else if cs.Properties.MasterProfile.Distro == AKS1804Deprecated {
+					cs.Properties.MasterProfile.Distro = AKSUbuntu1804
 				}
```

Comment:
remove this instead, if you want

Reply:
Removed in new PR: https://github.com/Azure/AgentBaker/pull/240